identity-service/src/notifications/notificationQueue.js

```diff
@@ -37,44 +48,71 @@ async function addNotificationToBuffer (message, userId, tx, buffer, playSound,
   buffer.push(bufferObj)
 }
 
+async function _sendNotification (notifFunctionName, bufferObj) {
+  const logPrefix = `[notificationQueue:sendNotification] [${notifFunctionName}] [userId ${bufferObj.userId}]`
+  try {
+    const start = Date.now()
+
+    await racePromiseWithTimeout(
+      SEND_NOTIF_FUNCTIONS[notifFunctionName](bufferObj),
```

Comment:
Not sure I love this pattern of using a singleton global object for these fn names. seems kinda error prone, why not just pass in the method?

_sendNotifications(sendAwsSns, bufferObj)

and then here do

await racePromiseWithTimeout(sendAwsSns(bufferObj), SEND_NOTIF_TIMEOUT_MS))

Reply:
just bc i needed the name in the log msg, but i can just pass in the name naively as a separate arg. agreed i don't love this either, will change